scripts/build/.variables

```diff
@@ -34,8 +20,59 @@ if [ "${GOARCH}" = "arm" ] && [ -n "${GOARM}" ]; then
 fi
 
 if [ "${GOOS}" = "windows" ]; then
-	TARGET="${TARGET}.exe"
+    TARGET="${TARGET}.exe"
```

Comment:
Looks like https://github.com/docker/cli/pull/2993 messed up formatting of out scripts and changed tabs to spaces for indentation 😞 

We should probably fix that again (perhaps add a `shfmt` check somewhere https://github.com/moby/moby/blob/d9e3d383f56119b0ba64ed63c6d5a265fbf35f7d/Dockerfile#L204-L209

https://github.com/moby/moby/blob/a77317882d010b884a9101c6ad0b2d7db141082f/hack/validate/shfmt#L4-L13

Reply:
Ah yes indeed, we should add shfmt in a follow-up, good idea.